index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Webpage</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+        <nav id="navbar">
+                <a href="#welcome-section">Welcome</a>
+                <a href="#projects">Projects</a>
+        </nav>
+    <section id="welcome-section">
+        <h1>Hello World! We are software developers.</h1>
+    </section>
```

Comment:
Please for uniformity and readability, maintain the same indentation for child elements and also make use of 2 spaces for the indentation to avoid linter error.